pkg/cmd/dedicated/listclusters/list.go

```diff
@@ -91,10 +91,10 @@ func listEnterpriseClusters(opts *options, f *factory.Factory) error {
 func createSearchString(opts *options) string {
 	searchString := ""
 	for idx, kfmcluster := range opts.kfmClusterList.Items {
-		searchString += fmt.Sprintf("id = '%s'", kfmcluster.Id)
-		if idx > 0 {
+		if idx >= 1 && idx < len(opts.kfmClusterList.Items) {
```

Comment:
will `idx` be ever equal to `len(opts.kfmClusterList.Items)`?

Reply:
Nope. Damn this is amateur hour over here

Reply:
for idx == 0 and `len(opts.kfmClusterList.Items)` == 0 🤔

Reply:
but in that case there are no items to range over so the loop never gets executed

Reply:
Yes. I wonder the usefulness of that check length check then.

Good thing is you've test - let's see what they say :-D